lgtm.yml

```diff
@@ -20,6 +20,3 @@ extraction:
       - dh-exec
       - doxygen
       - graphviz
-    after_prepare:
-      - git clone https://github.com/Azure/sonic-py-swsssdk.git; pushd sonic-py-swsssdk; python3 setup.py bdist_wheel; popd
```

Comment:
>clone [](http://example.com/codeflow?start=12&length=5)

swsssdk is still used in unit tests. So lgtm needs it.
Checking lgtm log, seem this PR build still `git clone`
```
[2022-06-28 21:16:56] [build-stderr] ++ git clone [https://github.com/Azure/sonic-py-swsssdk.git](https://github.com/Azure/sonic-py-swsssdk.git)
[2022-06-28 21:16:56] [build-stderr] Cloning into 'sonic-py-swsssdk'...
[2022-06-28 21:16:58] [build-stderr] ++ pushd sonic-py-swsssdk
[2022-06-28 21:16:58] [build-stdout] /opt/src/sonic-py-swsssdk /opt/src
[2022-06-28 21:16:58] [build-stderr] ++ python3 setup.py bdist_wheel
```

Reply:
Fixed, add removed lines back.